src/helpers/Url/Url.js

```diff
@@ -93,6 +93,20 @@ export function flattenToAppURL(url) {
     .replace(settings.apiPath, '');
 }
 
+/**
+ * Given a URL if it starts with the API server URL
+ * this method removes the /api or the /Plone part.
+ * @method toPublicURL
+ * @param {string} url URL of the object
+ * @returns {string} public URL
+ */
+export function toPublicURL(url) {
```

Comment:
Could be a separate setting. It's already added as a configurable in Plone via kitconcept.volto, right?

Reply:
sorry, right. We don't use kitconcept.volto and we dont'know it. We are adapting our backend to return the correct url for sitemap in robots.txt